_includes/footer.html

```diff
@@ -2,12 +2,12 @@
   <div class="wrapper">
 
     <nav class="footer-nav">
-      <a href="/demo" class="footer-link">Demo</a> &bull;
+      <a href="/blog/2016/06/why-hospitalrun" class="footer-link">Why HospitalRun?</a>&bull;
       <a href="https://github.com/hospitalrun" class="footer-link">Source</a> &bull;
       <a href="/events" class="footer-link" target="_blank">Hack Events</a> &bull;
       <a href="/blog" class="footer-link">Blog</a> &bull;
       <a href="/contribute" class="footer-link">Contribute</a> &bull;
-      <a href="/blog/2016/06/why-hospitalrun" class="footer-link">Why HospitalRun?</a>
+      <a href="/tryit" class="footer-link">Try it!</a>
```

Comment:
Let's keep the punctuation out of these links. It's tacky. Same for the `Why HospitalRun` link. The punctuation isn't needed.